plugin-volume/volumepopup.cpp

```diff
@@ -177,7 +183,9 @@ void VolumePopup::openAt(QPoint pos, Qt::Corner anchor)
 
 void VolumePopup::handleWheelEvent(QWheelEvent *event)
 {
-    m_volumeSlider->event(reinterpret_cast<QEvent*>(event));
+    int delta = event->angleDelta().y() / QWheelEvent::DefaultDeltasPerStep;
```

Comment:
As you reimplemented the `Slider::wheelEvent()`, this change is unnecessary.... calling the `m_volumeSlider->wheelEvent(event);` does the trick on it's own.